src/ralph/backends/data/clickhouse.py

```diff
@@ -91,8 +89,17 @@ class Config(BaseSettingsConfig):
     LOCALE_ENCODING: str = "utf8"
 
 
-class BaseClickHouseQuery(BaseQuery):
-    """Base ClickHouse query model."""
+class ClickHouseQuery(BaseModel):
+    """Base ClickHouse query model.
+
+    Attributes:
+        select:
+        where:
+        parameters:
+        limit:
+        sort:
+        column_oriented:
```

Comment:
```suggestion
    Attributes:
        select (str or list): Name of the table(s) to query.
        where (str or list): Expression for filtering the data.
        parameters (dict): Dictionary of substitution values.
        limit (int): Maximum number of rows returned.
        sort (str): Expression determining the sorting direction.
        column_oriented (bool): Whether to return the results as a sequence of columns rather than a sequence of rows.
```